nixos/modules/services/web-apps/writefreely.nix

```diff
@@ -258,6 +281,92 @@ in {
       };
     };
 
+    oauth = {
+      enable = lib.mkEnableOption "Enable generic OAuth authentication";
```

Comment:
mkEnableOption already adds a preamble to the description, so we can change this to just specify "generic OAuth".